modules/microshift-install-bootc-prepare-kickstart.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// microshift_install_bootc/microshift-install-running-bootc-image-in-VM.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-install-bootc-prepare-kickstart_{context}"]
+= Creating the Kickstart file
+
+You must create the Kickstart file to be used during installation by using the following procedure.
+
+.Prerequisites
+
+* You have root-user access.
+* You are logged in to the physical hypervisor host.
+
+.Procedure
+
+. Set the variables to reference the secret files in the `kickstart.ks` file to authenticate private container registry access by running the following command:
++
+[source,terminal]
+----
+AUTH_CONFIG=~/.quay-auth.json
+----
+
+. Set the variables to reference the secret files in the `kickstart.ks` file to authenticate the {OCP} registry access by running the following command:
++
+[source,terminal]
+----
+$ PULL_SECRET=~/.pull-secret.json
+----
+
+. Set the variable to reference the image mode container image to be used during installation by running the following command:
++
+[source,terminal,subs="attributes+"]
+----
+$ IMAGE_REF="quay.io/_<myorg>/<mypath>_/microshift-{product-version}-bootc" <1>
+----
+<1> Replace _<myorg/<mypath>_ with your remote registry organization name and path.
+
+. Create the `kickstart.ks` file to be used during installation by running the following command:
```

Comment:
```suggestion
. Create the `kickstart.ks` file to use during installation by running the following script:
```

This looks like a script or is this something that users will run as a single command?

Reply:
It is a script, yeah.